tests/integration/core/pg/01-deploy-operators.yaml

```diff
@@ -5,6 +5,6 @@ commands:
   - command: kubectl apply --server-side -f https://raw.githubusercontent.com/percona/percona-postgresql-operator/v${PG_OPERATOR_VERSION}/deploy/crd.yaml
   # We don't need the PG operator to be running for the integration tests, but we need the deployment to exist otherwise the everest operator won't reconcile the CRs
   - script: "curl -fsSL https://raw.githubusercontent.com/percona/percona-postgresql-operator/v${PG_OPERATOR_VERSION}/deploy/operator.yaml | sed 's/replicas: [0-9]/replicas: 0/g' | kubectl apply -n ${NAMESPACE} -f -"
-  - script: SYSTEM_NAMESPACE=$NAMESPACE MONITORING_NAMESPACE=$NAMESPACE DB_NAMESPACES=$NAMESPACE ../../../../bin/manager -metrics-bind-address :0 -health-probe-bind-address :0
+  - script: SYSTEM_NAMESPACE=$NAMESPACE MONITORING_NAMESPACE=$NAMESPACE DB_NAMESPACES=$NAMESPACE ../../../../bin/manager -metrics-bind-address :0 -health-probe-bind-address :0 -disable-webhook-server=true
     background: true
-    skipLogOutput: true
+    skipLogOutput: false
```

Comment:
Was this intentional or a leftover from debugging?

Reply:
not intentional, reverted